muni_portal/tests/api/test_service_page.py

```diff
@@ -25,17 +29,60 @@ def test_service_page_image(self):
             depth=2,
             profile_image=profile_image,
         )
-        page = ServicePage(
+        # create page here to be able to reuse it in next tests
+        cls.page = ServicePage(
             title="Test ServicePage",
             slug="test-service-page",
             path="00012222",
             depth=2,
             live=True,
             icon_classes=[],
+            office_hours=OFFICE_HOURS_TEST_TEXT,
             head_of_service=head_of_service
         )
-        Site.objects.first().root_page.add_child(instance=page)
-        response = self.client.get(self.url + f"{page.id}/")
-        assert response.status_code == status.HTTP_200_OK
-        assert response.json()["head_of_service"]["profile_image"]
-        assert response.json()["head_of_service"]["profile_image_thumbnail"]
+        cls.site.root_page.add_child(instance=cls.page)
+
+    def setUp(self):
+        self.client = Client()
+        self.url = reverse("wagtailapi:pages:listing")
+
+    def test_service_page_image(self):
+        response = self.client.get(self.url + f"{self.page.id}/")
+        res_as_dict = response.json()
+
+        # using `self.assertEqual` instead of `assert` here is more
+        # common and better way when using unittest
+        # NOTE: `assert` is good to use with py.test
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(
+            "profile_image" in res_as_dict["head_of_service"],
+            True
+        )
+        self.assertEqual(
+            "profile_image_thumbnail" in res_as_dict["head_of_service"],
+            True
+        )
+
+    def test_service_page_office_hours(self):
+        response = self.client.get(self.url + f"{self.page.id}/")
+        res_as_dict = response.json()
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual("office_hours" in res_as_dict, True)
+        self.assertEqual(res_as_dict["office_hours"], OFFICE_HOURS_TEST_TEXT)
+
+    def test_service_point_page_office_hours(self):
+        service_point_page = ServicePointPage(
```

Comment:
See above - how about the suffix `_serialization1 ?

```suggestion
    def test_service_point_page_serialisation(self):
        service_point_page = ServicePointPage(
```